tiledb/sm/cpp_api/domain.h

```diff
@@ -317,7 +317,7 @@ class Domain {
 
 /** Get a string representation of an attribute for an output stream. */
```

Comment:
Let's also fix this old typo in this comment: `an attribute` -> `the domain`.

Reply:
sure, I am running "make format" and "make check" again under centos7. It seems my unbuntu for windows has some problems with format checking